docs/guides/bonsai.md

```diff
@@ -0,0 +1,49 @@
+(bonsai)=
+
+# Using Bonsai with SLEAP
+
+Bonsai is a visual language for reactive programming and currently supports SLEAP models.
+
+**Exporting a SLEAP trained model:**
+
+:::{note}
+Currently Bonsai supports only single instance, top-down and top-down-id SLEAP models.
+:::
+
+In order to import the trained model into Bonsai, we need to export the model using the {code}`sleap-export` cli command to convert the trained model to a Protocol buffer(.pb) format. For example, for the top-down-id model, the command is as follows:
+
+```
+sleap-export -m centroid_model_folder_path -m top_down_id_model_folder_path exported_model
+```
+
+(for further details please refer {ref}`sleap-export` docs for more details.)
+
+This will generate the necessary `.pb` file and other information files to be used in Bonsai in the `exported_model` folder.
```

Comment:
```suggestion
This will generate the necessary `.pb` file and other information files required by Bonsai. In this example, these files were saved to the specified `exported_model` folder.
```